### PR TITLE
Fixing dialog CSS for RTL

### PR DESCRIPTION
**Pull request by antonlabunets**
Dialog and contents were previously still left-aligned in RTL. This fixes it. Tested by setting html dir to ltr and RTL.

### CODE REVIEW

**Comment by dzearing** [comment]
Approved! Thanks!

File: src/components/Dialog/Dialog.scss [hunk 1]
```diff
@@ -148,7 +148,8 @@
 
   // Spacing between bottom buttons
   .ms-Dialog-action + .ms-Dialog-action {
-    margin: 0 0 0 16px;
+    margin: 0;
+    @include margin-right(16px);
```
Comment (Jahnp) [inline src/components/Dialog/Dialog.scss]: I think this should be `@include margin-left` to be consistent with the original values.

**Comment by Jahnp** [comment]
Approved 🍂 

With a comment. Please address it before merging. Thanks so much for doing this!

File: src/components/Dialog/Dialog.scss (hunk 1 above)
Comment (antonlabunets) [inline src/components/Dialog/Dialog.scss]: Sorry, i mixed TRBL with TLBR. Coming up in an update!

**Comment by benkaiser** [comment]
@antonlabunets @dzearing has the dialog since regressed? (see X icon from docs pages with <html dir="rtl"> )
image

**Pull request merged**